pages/linux/qm-create.md

```diff
@@ -19,6 +19,6 @@
 
 `qm create {{100}} --archive {{path/to/backup_file.tar}} --force 1`
 
-- Specify a script that is executed on specific triggers during machine lifetime:
+- Specify a script that is automatically executed depending on the state of the virtual machine:
```

Comment:
```suggestion
- Specify a script that is executed automatically depending on the state of the virtual machine:
```
I think this would be better.

Reply:
I believe you could write it both ways, but reading it over a few times, I agree. Sounds a little bit more fluent.